src/demos/datepicker/fuzzy-datepicker-demo.component.html

```diff
@@ -0,0 +1,133 @@
+<h2
+  class="sky-section-heading"
+>
+  Standard fuzzy datepicker
+</h2>
+
+<div
+  style="max-width: 50%"
+>
+  <sky-datepicker>
+    <input
+      skyFuzzyDatepickerInput
+      [futureDisabled]="futureDisabled"
+      [dateFormat]="dateFormat"
+      [disabled]="disabled"
+      [maxDate]="maxDate"
+      [minDate]="minDate"
+      [skyFuzzyDatepickerInput]
+      [skyDatepickerNoValidate]="noValidate"
+      [startingDay]="startingDay"
+      [yearRequired]="yearRequired"
+      [(ngModel)]="selectedDate"
+      #date="ngModel"
+    />
+  </sky-datepicker>
+
+  <p *ngIf="date.errors && date.errors.skyFuzzyDate && (date.dirty || date.touched)"
+    class="sky-error-label"
+  >
+    <span *ngIf="date.errors.skyFuzzyDate?.invalid">
+      Please enter a valid date.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.yearRequired">
+      Please select a year.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.futureDisabled">
+      Date cannot be in the future.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.maxDate">
+      The current date is beyond the maximum allowed date.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.minDate">
+      The current date is before the minimum allowed date.
+    </span>
+  </p>
+</div>
+
+<p>
+  Selected fuzzy date is {{ selectedDateForDisplay }}
+</p>
+
+<h2
+  class="sky-section-heading"
+>
+  Reactive form fuzzy datepicker
+</h2>
+
+<form
+  novalidate
+  style="max-width: 50%"
+  [formGroup]="reactiveForm"
+>
+  <sky-datepicker>
+    <input
+      formControlName="selectedDate"
+      skyFuzzyDatepickerInput
+      [dateFormat]="dateFormat"
+      [maxDate]="maxDate"
+      [minDate]="minDate"
+      [skyFuzzyDatepickerInput]
+      [skyDatepickerNoValidate]="noValidate"
+      [startingDay]="startingDay"
+      [yearRequired]="yearRequired"
+    />
+  </sky-datepicker>
+</form>
+
+<p>
+  Reactive form selected fuzzy date is {{ reactiveFormSelectedDateForDisplay }}
```

Comment:
As with the corresponding text above for the standard fuzzy datepicker, this is currently a sentence fragment with no punctuation at the end. Also, do we really need "Reactive form" at the beginning to distinguish this from the previous text? I don't think so; I think placement on the page makes that clear, I'd delete "Reactive form." And then I'd either capitalize "Selected," delete "is," and add a colon after "date:" to make it more like a key-value pair, or else I'd add "The" to the start and add a period to the end to make this a complete sentence.

Reply:
Done.

Reply:
Ditto previous messages on date format.

Reply:
Done.